src/coreservices.cpp

```diff
@@ -423,6 +423,29 @@ void CoreServices::initialize(QApplication* pApp) {
 
     m_pTouchShift = std::make_unique<ControlPushButton>(ConfigKey("[Controls]", "touch_shift"));
 
+    // The following wmainmenubar controls must be created here so that controllers can bind to them
+    // on startup.
+    m_pSkinSettingsControl = std::make_unique<ControlObject>(
+            ConfigKey("[Master]", "skin_settings"));
```

Comment:
m_pSkinSettingsControl->setButtonMode(ControlPushButton::TOGGLE);

Please add this for all of them. This means the state can be toggled after learning them via midi. Else you need to hold the button for seeing the preferences.